src/async_fuse/memfs/cache/mod.rs

```diff
@@ -407,26 +411,24 @@ impl GlobalCache {
         let read_fn = |global_index: usize| -> IoMemBlock {
             let hash_index = global_index.overflow_div(bucket_size);
             let bucket_opt = file_cache.get(&hash_index, &guard);
-            /*
-            if let Some(bucket) = bucket_opt {
-                if let Some(block) = bucket
-                    .write()
-                    .get(global_index.overflow_rem(bucket_size))
-                    .unwrap_or_else(|| {
-                        panic!(
-                            "error when getting range of {} in the cache bucket",
-                            global_index.overflow_rem(bucket_size)
-                        )
-                    })
-                {
-                    IoMemBlock::new(Some(block.clone()), 0, block_size)
-                } else {
-                    IoMemBlock::new(None, 0, 0)
-                }
-            } else {
-                IoMemBlock::new(None, 0, 0)
-            }
-            */
+            // if let Some(bucket) = bucket_opt {
```

Comment:
Same as above. And if we really want to comment some code and keep the indent, is there any way to work around.

Reply:
Why are they still here and unresolved?